app/policies/article_policy.rb

```diff
@@ -2,28 +2,71 @@
 
 class ArticlePolicy < ApplicationPolicy
   def index?
-    user.root? or user.editor?
+    user.root? or user.editor? or user.neighbourhood_admin? or user.partner_admin?
+    return true if user.root? or user.editor?
+    return true if user.partner_admin? && user.partners.count.positive?
+
+    # True if neighbourhood admin oversees any partners
+    return true if user.neighbourhood_admin? && owned_neighbourhoods_have_partners?
   end
 
-  def edit?
-    user.root? or user.editor?
+  def show?
+    index?
+  end
+
+  def create?
+    index?
   end
 
   def new?
-    user.root? or user.editor?
+    index?
   end
 
-  def create?
-    user.root? or user.editor?
+  def update?
+    index?
+  end
+
+  def edit?
+    update?
   end
 
   def destroy?
-    user.root? or user.editor?
+    update?
+  end
+
+  def permitted_attributes
+    %i[title body published_at is_draft].push(partner_ids: [])
```

Comment:
finding this mixed array notation a bit weird not sure its this but a tiny bit of a code smell

Reply:
alternative is `[:title, :body, :published_at, :is_draft, partner_ids: []]`. I think I prefer the `.push` stylistically because it means it's easier to add other elements, but can move it over :)

Reply:
fixed as of latest push